pokedex/src/main/java/hive/pokedex/exception/UsernameAlreadyExistsException.java

```diff
@@ -0,0 +1,8 @@
+package hive.pokedex.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.CONFLICT, reason = "Username already registered, try again")
```

Comment:
```suggestion
@ResponseStatus(code = HttpStatus.CONFLICT, reason = "Username already registered, try register other username")
```
"try again" da impressão que deve tentar de novo com os mesmo dados.